### PR TITLE
Code refactor and enhancement

### LINKED ISSUE #60: Mongo Query to get quotes by tags with author detail
update mongodb query to filter quotes by tags with author name and id detail.

### PR DESCRIPTION
**Pull request by karelumair**
- [X] Updated mongo query to get quotes by tags along with author detail
- [X] Fixed issue with scarping quotes text formatting.
- [X] Fixed issues in with author id in Quotes POST API

closes #60

### CODE REVIEW

File: utils/scraping.py [hunk 1]
```diff
@@ -31,7 +32,8 @@ def scrape_quotes(self, url) -> dict:
     while next_page:
 
         for quote_div in driver.find_elements(By.CLASS_NAME, "quote"):
-            quote = quote_div.find_element(By.CLASS_NAME, "text").text
+            quote = quote_div.find_element(By.CLASS_NAME, "text")
+            quote = re.sub(r"[\“\”]", "", quote.text)
```
Comment (pratikGhodke1) [inline utils/scraping.py:39]: Can you add a comment and a short example here for this code statement?

**Review by pratikGhodke1 — CHANGES_REQUESTED** [review]
Add a comment as suggested in review comment

**Comment by karelumair** [comment]
> Add a comment as suggested in review comment

Done.

**Review by pratikGhodke1 — APPROVED** [review]

**Comment by pratikGhodke1** [comment]
Don't add separate comment for `adding comment`. Add those changes along with other changes as refactor change.

**Pull request merged**